### ISSUE TITLE
BBS reporting warning after data changes

### ISSUE THREAD

**Issue by ethanwhite** [weecology/retriever#460]
At the end of a BBS install using #459 the following warnings are reported:

```
Inserting data from Quebec...
Inserting data from Saskatchewan...
Inserting data from Yukon...
WARNING (BBS:species): Exception in line 1: invalid literal for int() with base 10: 'Seq'
WARNING (BBS:species): Exception in line 2: invalid literal for int() with base 10: '------'
Done!
```

**Comment by henrykironde** [comment]
@ethan which database engine are you using

**Comment by ethanwhite** [comment]
That was with sqlite: `retriever install sqlite BBS -f bbs.sqlite`

**Comment by henrykironde** [comment]
is any one still getting this error?

**Comment by ethanwhite** [comment]
Fixed in ec7a0719bc797bd96e260e3d78d454ba0c293864.
